# Generated by Django 4.0.4 on 2022-05-10 05:26

import datetime
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('manager', '0013_reason_for_refund_refusal_or_refund_status_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='external_transfer',
            name='transfer_date',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 8, 26, 19, 995394), null=True),
        ),
        migrations.AlterField(
            model_name='internal_transfer',
            name='transfer_date',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 8, 26, 19, 997396), null=True),
        ),
        migrations.AlterField(
            model_name='profile_specialist',
            name='date_create',
            field=models.DateTimeField(default=datetime.datetime(2022, 5, 10, 8, 26, 19, 998398), null=True),
        ),
        migrations.CreateModel(
            name='Service_cancellation',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('client', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.customuser')),
                ('reason', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.reason_for_refund')),
                ('status', models.ForeignKey(on_delete=django.db.models.deletion.RESTRICT, to='manager.refusal_or_refund_status')),
            ],
        ),
    ]
